_posts/2016-06-20-alpha-release-notes.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "argparse2cwl alpha release notes"
+date:   2016-06-20 02:27:27 +0300
+comments: true
+---
+
+`argparse2cwl` forms CWL tool descriptions from programs which use `argparse` as their argument parser. 
+
+## Prerequisites ##
+`argparse2cwl` is compatible with both Python 2 & 3.
+Dependencies:
+* Jinja2 >= 2.8
+
+## Installing ##
+
+Installation process:
+
+1. Clone the source repository:
```

Comment:
It would be worth publishing this repo under PyPi to ease the install/release process... you can try to publish it under test.pypi:

https://testpypi.python.org/pypi

You can either use some tool/process to release it:

https://github.com/michaeljoseph/changes
https://seed.readthedocs.io/en/latest/

Or just release it yourself:

https://wiki.python.org/moin/CheeseShopTutorial

That lowers the barrier of entry for new users/developers ;)